DSharpPlus/DiscordClient.cs

```diff
@@ -2438,6 +2447,19 @@ internal async Task SendIdentifyAsync(StatusUpdate status)
             };
             var payloadstr = JsonConvert.SerializeObject(payload);
             await this._webSocketClient.SendMessageAsync(payloadstr).ConfigureAwait(false);
+
+            if (this.Configuration.Intents.HasValue)
+            {
+                var intentList = Enum.GetValues(typeof(DiscordIntents)).Cast<DiscordIntents>().ToArray();
+                for (int i = 0; i < intentList.Length; ++i)
+                {
+                    if (intentList[i] == DiscordIntents.All || intentList[i] == DiscordIntents.AllUnprivileged)
+                        continue;
+
+                    if (this.Configuration.Intents.Value.HasIntent(intentList[i]))
+                        this.DebugLogger.LogMessage(LogLevel.Debug, "DSharpPlus", $"Registered intent: {intentList[i]}.", DateTime.Now);
+                }
+            }
```

Comment:
Is this only for logging? If it is, probably not required.

Reply:
Yeah I was debating this as well. I ultimately chose to go with it because of useful info, and also that this doesn’t really hurt the lib.

Reply:
Yeah I was debating this as well. I ultimately chose to go with it because of useful info, and also that this doesn’t really hurt the lib.

Reply:
True. I had a for loop being faster in mind, but in this case I suppose it doesn’t really make a difference.